### PR TITLE
Rewrite homepage to align with designs

### PR DESCRIPTION
**Pull request by cmyui**
Does not include score carousel yet

### CODE REVIEW

File: src/components/images/banners/homepage_banner.png
Comment (cmyui) [inline src/components/images/banners/homepage_banner.png:1]: todo swap to mp3

File: src/pages/HomePage.tsx [hunk 1]
```diff
@@ -1,85 +1,131 @@
...
-    </Grid>
+    <Box>
+      {/* TODO: scale the text outline with screen size */}
+      <Box position="absolute" zIndex={0} top={0} left={0} width="100%">
+        <HomepageTextOutline />
+      </Box>
+      <Stack direction="column" justifyContent="space-between" spacing={2}>
+        <Box
+          pt={{ xs: 0, sm: 16 }}
+          pb={{ xs: 0, sm: 16 }}
```
Comment (cmyui) [inline src/pages/HomePage.tsx:23]:
```suggestion
          py={{ xs: 0, sm: 16 }}
```

File: src/pages/HomePage.tsx [hunk 2]
```diff
@@ -1,85 +1,150 @@
...
+        <Box
+          py={{ xs: 0, sm: 16 }}
+          sx={{
+            backgroundImage: `url(${HomepageBanner})`,
+            backgroundSize: "cover",
+            backgroundPosition: "bottom",
+          }}
+        >
+          <Container>
+            <Grid
```
Comment (infernalfire72) [inline src/pages/HomePage.tsx:41]: is there any reason we're using grid instead of flexbox here?

File: src/pages/HomePage.tsx [hunk 3]
```diff
@@ -1,85 +1,150 @@
...
+              spacing={2}
+              justifyContent="center"
+            >
+              <Grid item xs={6}>
+                <Stack
+                  direction="column"
+                  alignItems={{ xs: "center", sm: "flex-end" }}
+                  spacing={3}
+                >
+                  <Button>
```
Comment (infernalfire72) [inline src/pages/HomePage.tsx:53]: why is this a button lmao

**Pull request merged**